user_guide.adoc

```diff
@@ -3090,19 +3089,21 @@ the_rest = ".bar."
 [[eos-error]]
 ==== Ignoring errors in delimited structures
 
-Delimited structures actually could be pretty scary: if we read until
-the terminator is encountered, what will happen if we'll never find
-one? In raw C, a typical answer is "segmentation fault" occurring in
-the operation that deals with such string due to runaway reads past
+Delimited structures actually can be pretty scary: if we read until
+the terminator is encountered, what will happen if we never find
+one? In raw C, a typical answer is a "segmentation fault" occurring in
+the operation that deals with such strings due to runaway reads past
 the buffer, i.e. a very bad thing and a big security hole.
 
 In Kaitai Struct, however, you can control this behavior. By default,
-if you'll parse a structure awaiting for a terminator, and that
+if you parse a structure awaiting a terminator, and that
 terminator never happens, you'll get a clear error / exception similar
-to "attempted to read past end of stream". However, in some cases, you
-really want to read until the end of stream, and this should be
-considered normal encoding, not an error. For this cases, you can
-specify `eos-error`:
+to "attempted to read past end of stream".
+
+In some cases, you
+really want to read until the end of the stream, and this should be
+considered normal encoding, not an error. For these cases, you can
+specify `eos-error: false` (the default is true):
```

Comment:
I'd go for

```suggestion
specify `eos-error: false` (the default is `true`):
```

... so it has the same typeface as the value `false` a few words before.